Tribes/src/app/_components/progress-bar/progress-bar.component.ts

```diff
@@ -33,9 +38,9 @@ export class ProgressBarComponent implements OnInit {
     return this.result = 100 - ((minuteDiff / totalMinutes) * 100);
   }
 
-  // refreshProgressBar(): void {
-  //   this.buildingService.beginConstruction.emit();
-  //   setInterval(() => {this.checkTimeDifference(this.notification)}, 1000);
-  // }
+  refreshProgressBar(): void {
+    this.buildingService.beginConstruction.subscribe();
```

Comment:
Remove this line, the subscription is in the constructor, no need to subscribe again ;) 
Otherwise 🥇